hal/src/main/native/athena/REVPDH.cpp

```diff
@@ -35,6 +36,7 @@ struct REV_PDHObj {
   int32_t controlPeriod;
   HAL_CANHandle hcan;
   std::string previousAllocation;
+  HAL_PowerDistributionVersion versionInfo;
```

Comment:
You'll need to explicitly zero this out. It wont zero by default.

Reply:
Same with the ph ones

Reply:
Then why doesn't `desiredSolenoidsState` on the PH object get explicitly zeroed out anywhere? That seems to work fine, and it definitely is written to assume a zeroed-out starting point. Am I missing something there?

Reply:
Oh thats another bug.

Reply:
Since its close to program initialization, its likely to be 0'd, but no guarantees.

Reply:
Gotcha, thanks.

Reply:
`solenoidLock` too, or is that different because it's a proper C++ object instead of a struct?

Reply:
That one's fine because it's a c++ object